library/Isochrone_IGN_API.py

```diff
@@ -95,11 +99,12 @@ def request_IGN_isochrone_api(point:str , costValue:int, resource:str='bdtopo-va
         call=requests.get('https://data.geopf.fr/navigation/isochrone',params=api_body,headers=api_headers)
         call.raise_for_status()
         
-        gdf = gpd.GeoDataFrame(
-            [{'geometry': shape(call.json()['geometry']), **{k: v for k, v in call.json().items() if k != 'geometry'}}],
-            geometry='geometry',
-            crs='EPSG:4326'
-        )
+        base_dict = {'geometry': shape(call.json()['geometry']), **{k: v for k, v in call.json().items() if k != 'geometry'}}
+
+        if valueKey is not None and valueKey != 'None':
+            base_dict['valueKey'] = valueKey
+
+        gdf = gpd.GeoDataFrame([base_dict], geometry='geometry', crs='EPSG:4326')
```

Comment:
```suggestion
        df = gpd.GeoDataFrame([{'geometry': shape(call.json()['geometry']), **{k: v for k, v in call.json().items() if k != 'geometry'}}], geometry='geometry', crs='EPSG:4326')
```

Returned to the previous code because I've moved valueKey out of the function.